doc.sh

```diff
@@ -7,6 +7,25 @@ pushd manual
 bash run.sh
 popd
 
+# compile openrpc cli binary
+v ~/.vmodules/freeflowuniverse/crystallib/openrpc/cli
+OPENRPC_CLI=~/.vmodules/freeflowuniverse/crystallib/openrpc/cli/cli
+
+# generate doc for entire api
+$OPENRPC_CLI docgen -t "Web3Proxy JSON-RPC API" -p -o server lib
```

Comment:
We still need that if we do them individually?

Reply:
not really, but why not. I link to it's playground in the ui, maybe it interests some people to see the full scope of the api. I'm ok with removing it

Reply:
Ok. Let's keep it until we have a reason not too